scripts/set-alias-page.py

```diff
@@ -6,17 +6,19 @@
 
 Disclaimer: This script generates a lot of false positives so it
 isn't suggested to use the sync option. If used, only stage changes
-and commit verified changes for your language.
+and commit verified changes for your language by using -l LANGUAGE.
 
 Note: If there is a symlink error when using the stage flag remove the `pages.en`
 directory temporarily and try executing it again.
 
 Usage:
-    python3 scripts/set-alias-page.py [-p PAGE] [-s] [-S] [-n] [COMMAND]
+    python3 scripts/set-alias-page.py [-p PAGE] [-l LANGUAGE] [-s] [-S] [-n] [COMMAND]
 
 Options:
     -p, --page PAGE
         Specify the alias page in the format "platform/alias_command.md".
+    -l, --language LANGUAGE
+        Specify the language, in the format "fr" or "pt_BR", for which you want to sync or show any changes.
```

Comment:
```suggestion
        Specify the language, a POSIX Locale Name in the form of "<2_letter_language>[_<2_LETTER_CONTRY>]" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

If you think this is too big/complex, I have another suggestions:

```suggestion
        Specify the language, in the form of "ll" or "ll_CC" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

```suggestion
        Specify the language, in the form of "locale" or "locale_COUNTRY" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

```suggestion
        Specify the language, a POSIX Locale Name in the form of "<language>[_<COUNTRY>]" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

```suggestion
        Specify the language, a POSIX Locale Name in the form of "ll" or "ll_CC" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```